perf/istio-install/setup_prometheus.sh

```diff
@@ -14,18 +14,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -ex
+set -x
```

Comment:
I hate bash.. its useful to have `set` in most places, but I am not sure how we can do the retry loop with it, so I guess its ok to turn off here